d2l-icon-button-behavior.html

```diff
@@ -1,10 +1,21 @@
 <script>
-	window.D2L.PolymerBehaviors = window.D2LBehaviors || {};
-	window.D2L.PolymerBehaviors.IconButtonBehavior = {
+(function() {
+	'use strict';
+
+	/** @polymerBehavior */
```

Comment:
@Surekh : something to be aware of - these "@polymerBehavior" comments are required for polylint to correctly lint.